### ISSUE TITLE
New ORT run details page(s)

### ISSUE THREAD

**Issue by Etsija** [eclipse-apoapsis/ort-server#680]
The run details page needs a complete overhaul. The new structure will probably be:

`organizations/{orgId}/products/{productId}/repositories/{repoId}/runs/{runIndex}`:
- [x] card layout. Example of such [is here](https://ui.shadcn.com/blocks) #706 
- [x] general run information #706
- [ ] issues list, pending on #405 
- [ ] job details, using a tabbed layout which will be reachable with search parameter `.../runs/{runIndex}?job={jobName}`
- [x] side menu (for example with icons such as in the above example): Overview, Logs, Reports

`organizations/{orgId}/products/{productId}/repositories/{repoId}/runs/{runIndex}/logs`:
- [x] logs of the run: zipped
- [ ] logs of the run: streaming

`organizations/{orgId}/products/{productId}/repositories/{repoId}/runs/{runIndex}/reports`:
- [x] downloadable reports from the run #770 

As this will be a big issue, the best way to implement it would be to use a temporary route `.../runs2/{runIndex}` with its subroutes for the implementation.  This way, we do the implementation as added pages and components and avoid any possible rebasing problems. We could possibly do intermediate smaller PRs to `main`, to expose the implementation in `.../runs2/` for all users to comment, while keeping the server UI otherwise intact (so all links still point to the old details page).

- [ ] In the end, the new pages will be migrated into the `.../runs` route.
- [ ] Clicking a traffic light for a run in repository index page should open up the new details page with the corresponding job details tab open, #643

**Comment by mnonnenmacher** [comment]
Some ideas I wrote down during the frontend meeting today:

- show badge next to menu items, e.g., for vulnerabilities, show the number of vulnerabilities (and mabye by severity, with different badge colors)
- move issues to separate page
- move job details to separate page (in technical section)
	- traffic lights in runs list could link to technical section with right tool selected
- show only summaries in overview page, like number of issues by severity
- show some statistical data in the overview page, e.g. a pie chart with number of projects per package manager
- show item next to menu items where data is still being generated, e.g. a spinner if the evaluator job is still running, or another item if it did not even start yet
- maybe show traffic lights in overview page, because users know it from the ort runs list

**Comment by Etsija** [comment]
@mnonnenmacher I decided to make a new "improvement" issue for the new results pages, as they become fully visible now, and create the issue in a checklist format. This issue contains ideas from a while back, of which many are now outdated, so I think it would be best to close this with the "runs2" merge and continue tracking on the new issue, where it's easy to even create issues out of the checklist items. I already put the contents of the alignment meeting to the new issue.
